interfaces/builtin/pluggable_storage.go

```diff
@@ -0,0 +1,40 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2016 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package builtin
+
+import (
+	"github.com/snapcore/snapd/interfaces"
+)
+
+const pluggableStorageConnectedPlugAppArmor = `
+# Description: Can access removable storage filesystems
+
+# Filesystems are mounted to /run/media/<user>/foo since /media is read-only
+/run/media/** rw,
+`
+
+// NewPluggableStorageInterface returns a new "pluggable-storage" interface.
+func NewPluggableStorageInterface() interfaces.Interface {
+	return &commonInterface{
+		name: "pluggable-storage",
+		connectedPlugAppArmor: pluggableStorageConnectedPlugAppArmor,
+		reservedForOS:         true,
```

Comment:
What is the indented use case? This will mean that hypothetical udisks2 snap won't be able to have a slot of this type. I was expecting that it would always have permissions simply because of the slot presence.